public/about-us.html

```diff
@@ -0,0 +1,288 @@
+                <div class="col-md-3">
+                    <div class="card card-profile card-plain">
+                        <div class="card-avatar">
+                            <a href="#avatar">
+                                <img class="black-white-img" src="./assets/img/avatars/cclaessens.jpeg" alt="...">
+                            </a>
+                        </div>
+                        <div class="card-body pt-0">
+                            <a href="#paper-kit">
+                                <div class="author">
+                                    <h4 class="avatar-card-title">Caroline Claessens</h4>
+                                    <h6 class="text-muted">Legal Officer</h6>
+                                </div>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-3">
+                    <div class="card card-profile card-plain">
+                        <div class="card-avatar">
+                            <a href="#avatar">
+                                <img class="black-white-img" src="./assets/img/avatars/adekker.jpeg" alt="...">
+                            </a>
+                        </div>
+                        <div class="card-body pt-0">
+                            <a href="#paper-kit">
+                                <div class="author">
+                                    <h4 class="avatar-card-title">Andre Dekker</h4>
+                                    <h6 class="text-muted">Chief Scientific Officer</h6>
+                                    <h6 class="text-muted">Founder</h6>
+                                </div>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-3">
+                    <div class="card card-profile card-plain">
+                        <div class="card-avatar">
+                            <a href="#avatar">
+                                <img class="black-white-img" src="./assets/img/avatars/thendriks.jfif" alt="...">
+                            </a>
+                        </div>
+                        <div class="card-body pt-0">
+                            <a href="#paper-kit">
+                                <div class="author">
+                                    <h4 class="avatar-card-title">Tim Hendriks</h4>
+                                    <h6 class="text-muted">Software Engineer</h6>
+                                </div>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="row justify-content-center">
+                <div class="col-md-3">
+                    <div class="card card-profile card-plain">
+                        <div class="card-avatar">
+                            <a href="#avatar">
```

Comment:
@tim-hendriks is the link (`<a href="">`) needed for the styling? It now implies you can click on the name or image and get additional information.

Reply:
I'll check, don't think it's needed. It's a leftover from an older 'teams' section that I took from a commit a couple of years back.